spec/unit/knife/osc_user_reregister_spec.rb

```diff
@@ -19,7 +19,7 @@
 require "spec_helper"
 
 # DEPRECATION NOTE
-# This code only remains to support users still	operating with
+# This code only remains to support users still  operating with
```

Comment:
these double space autocorrects are funky, but whatevs

Reply:
yeah they're tabs->spaces.  less awful than having hard tab stops there.  will block them in the future.